docs/reference/mapping/fields/routing-field.asciidoc

```diff
@@ -120,7 +125,7 @@ expense of having to search more shards per request.
 
 When this setting is present, the formula for calculating the shard becomes:
 
-    shard_num = (hash(_routing) + hash(_id) % routing_partition_size) % num_primary_shards
+    shard_num = (hash(_routing) + hash(_id) % routing_partition_size) % ( num_routing_shards / routing_factor )
```

Comment:
This also needs the parentheses to be adjusted:
```
shard_num = ((hash(_routing) + hash(_id) % routing_partition_size) % num_routing_shards) / routing_factor
```

The formula is actually getting pretty long, I had to stare at it a while to make sure I understood. I wonder if we could break it up into two:
```
routing_value = hash(_routing) + hash(_id) % routing_partition_size
shard_num = (routing_value %  num_routing_shards) / routing_factor
```

The second calculation matches the one above, which might make it easier to follow.

Reply:
Great suggestion.